warehouse/query-core/src/test/java/datawave/query/attributes/DocumentTest.java

```diff
@@ -0,0 +1,307 @@
+package datawave.query.attributes;
+
+import com.google.common.collect.Sets;
+import datawave.query.util.TypeMetadata;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.apache.hadoop.io.Text;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class DocumentTest {
+    private final Key testKey1 = new Key(new Text("20000101_69"), new Text("foo%00;d8zay2.-3pnndm.-anolok"), new Text(), new ColumnVisibility("ALL"),
```

Comment:
Same comment as below, extract common parts to variables

Reply:
Updated as requested